src/diffusers/pipelines/stable_diffusion/pipeline_flax_stable_diffusion.py

```diff
@@ -389,7 +385,11 @@ def __call__(
 
 
 # TODO: maybe use a config dict instead of so many static argnums
-@partial(jax.pmap, static_broadcasted_argnums=(0, 4, 5, 6, 7, 9))
+@partial(
+    jax.pmap,
+    in_axes=(None, 0, 0, 0, None, None, None, 0, 0, 0),
```

Comment:
Very cool!

`in_axes` is 0 for non-static args, no ? Also, same comment as above, let's put some comments here.